mkdocs.yml

```diff
@@ -24,12 +24,12 @@ extra:
   social:
     - icon: fontawesome/brands/github
       link: https://github.com/zarhus
-    # - icon: fontawesome/brands/mastodon
-    #   link: https://fosstodon.org/@TBD
-    # - icon: fontawesome/brands/twitter
-    #   link: https://twitter.com/TBD
-    # - icon: fontawesome/brands/reddit
-    #   link: https://www.reddit.com/r/TBD
+      # - icon: fontawesome/brands/mastodon
+      #   link: https://fosstodon.org/@TBD
+      # - icon: fontawesome/brands/twitter
+      #   link: https://twitter.com/TBD
+      # - icon: fontawesome/brands/reddit
+      #   link: https://www.reddit.com/r/TBD
```

Comment:
If you uncomment those lines (`# `) then they'll have wrong indent.
pre-commit doesn't complain unless there is also another error then it'll show warning so we could leave it as it was?

Reply:
> If you uncomment those lines (# ) then they'll have wrong indent

Then, when uncommenting, the indent should be adjusted, `yamllint` requires starting space in a comment. 

> pre-commit doesn't complain unless there is also another error then it'll show warning so we could leave it as it was?

I saw the warning - I fixed the warning. Why leaving it?